src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1415,7 +1415,12 @@ function setupFsCopyModal() {
                 for (var i = 0 ; i < coursesTable.length; i++) {
                     htmlToAppend += "<div class=\"checkbox\">";
                     htmlToAppend += "<label><input type=\"checkbox\" name=\"copiedcoursesid\"";
-                    htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + coursesTable[i].id + "] : " + coursesTable[i].name;
+                    if (String(coursesTable[i].id) == courseid) {
+                    	htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + "<span class=\"text-color-red\">" + coursesTable[i].id + "</span>" + "] : " + coursesTable[i].name;
+                    	htmlToAppend += "<br><span style=\"font-size:11px\" class=\"text-color-red\">{Session currently in this course}</span>";
```

Comment:
Since we are using the bootstrap library, I would recommend using the ".small" class that it provides if it looks fine. (see "small text" in http://getbootstrap.com/css/)